src/components/Product/Product.js

```diff
@@ -0,0 +1,95 @@
+import React, { useState } from 'react';
+import firebase from '@firebase/app';
+function Product() {
+  const [option, setOption] = useState('');
+  const [data, setData] = useState({
+    name: '',
+    lastDate: '',
+  });
+  const addProduct = (event) => {
+    event.preventDefault();
+    console.log('send data...' + data.name, data.lastDate, option);
+    firebase
+      .firestore()
+      .collection('products')
+      .add({
+        name: data.name,
+        lastDate: data.lastDate,
```

Comment:
Per our Acceptance Criteria (AC), this should be `null` in case we don't set this value in the form. Right now, it's always a string (empty or with value)

![](https://cdn.zappy.app/27150c8a1b7c52a16fa6e29e076c2a5e.png)

> The last purchased date should be set to null initially (i.e. you can create an item without purchasing it)

Reply:
Hi Alejo, thak you,  we will make the change. :100:

Reply:
Thnaks, Is corrected :)